src/steps/command/plugins.ts

```diff
@@ -12,6 +12,8 @@ export class Plugin {
 
 export interface Plugins<T> {
     add(plugin: Plugin): T;
+    /** Return a list of plugins that match the given predicate */
+    filter(predicate: (plugin: Plugin) => boolean): Plugin[];
```

Comment:
can we just reuse the original signature here? E.g.:

```suggestion
    filter: Plugin[]['filter'];
```

Reply:
I didn't know you could do this 😮  Will add.